default.build

```diff
@@ -319,10 +319,21 @@
 			<in>
 				<items>
 					<include name="${nuget.nupackages.dir}/*.nupkg"/>
+					<exclude name="${nuget.nupackages.dir}/*.symbols.nupkg"/>
 				</items>
 			</in>
 			<do>
-				<echo message="nuget push ${filename} ${environment::newline()}" file="${nuget.nupackages.pushbatfile}" append="true"/>
+				<echo message="nuget push -source https://api.nuget.org/v3/index.json ${path::get-file-name(filename)} ${environment::newline()}" file="${nuget.nupackages.pushbatfile}" append="true"/>
```

Comment:
As I recall: `https://nuget.org/` should be enough to push. Also, if it is just nuget.org the symbols would be pushed automatically.

Reply:
I will test this on NHibernate.Caches.

Reply:
It does not push symbols package. This feature has been removed. NuGet/Home#6113